### PR TITLE
merge italia_rtd modules

### PR DESCRIPTION
**Pull request by yakky**
Moves italia_rtd modules in docsitalia

### CODE REVIEW

File: readthedocs/docsitalia/resolver.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+class ItaliaResolver(ResolverBase):
+
+    """Custom path resolver for built documentation
+
+    Resolves to public domain without use of subdomains or /doc/* paths
+    """
+
+    def base_resolve_path(self, project_slug, filename, version_slug=None,
```
Comment (yakky) [inline readthedocs/docsitalia/resolver.py]: This method must be cleaned up a bit

File: readthedocs/docsitalia/signals.py [hunk 2]
```diff
@@ -32,3 +33,13 @@ def on_project_import(sender, **kwargs): # noqa
     )
     for pub_proj in pub_projects:
         pub_proj.projects.add(project)
+
+
+@receiver(finalize_sphinx_context_data)
+def add_sphinx_context_data(sender, **kwargs):
+    data = kwargs.get('data')
+    build_env = kwargs.get('build_env')
+    subprojects = (apiv2.project(build_env.project.pk)
```
Comment (yakky) [inline readthedocs/docsitalia/signals.py]: This method must be cleaned up a bit

File: readthedocs/docsitalia/resolver.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+"""Override RTD URL resolver"""
+
+from django.conf import settings
+
+from readthedocs.core.resolver import ResolverBase
+
+
+class ItaliaResolver(ResolverBase):
+
+    """Custom path resolver for built documentation
```
Comment (xrmx) [inline readthedocs/docsitalia/resolver.py]: The linter will probably want you to add a new line

**Review by xrmx — COMMENTED** [review]
Before cleaning we must add some tests please, even shallow one just to avoid typos

**Comment by yakky** [comment]
@xrmx I would need some help to get the tests running. I'm sure there is something wrong in my environment and I can't get them to run. Even with no changes applied on top of the starting branch

File: readthedocs/settings/testdocsitalia.py [hunk 4]
```diff
@@ -13,10 +13,19 @@
     }
 }
 
+CLASS_OVERRIDES = {
```
Comment (xrmx) [inline readthedocs/settings/testdocsitalia.py]: To fix tests I'd avoid these changes here and add them with override_settings on our own tests

Comment (yakky) [reply readthedocs/settings/testdocsitalia.py]: yes, that was a last ditch-attempt 
settings_override does not work with `CLASSES_OVERRIDES` anyway, thus to test the resolver I think the best way is just to test that specific method

**Review by xrmx — APPROVED** [review]

**Pull request merged**